tests/clvm/test_singletons.py

```diff
@@ -87,7 +87,7 @@ async def test_singleton_top_layer(self, version):
             # START TESTS
             # Generate starting info
             key_lookup = KeyTool()
-            pk: G1Element = public_key_for_index(1, key_lookup)
+            pk: G1Element = G1Element.from_bytes(public_key_for_index(1, key_lookup))
```

Comment:
I would ask that the function be hinted so it will get checked to catch stuff like this...  but that exposes 72 complaints...  Another time I guess.

Reply:
you mean G1Element.from_bytes? We should def. hint the bls stuff

Reply:
No, the test function.  Sorry.  But yes, everything should get hinted.  `:]`